Cargo.toml

```diff
@@ -31,9 +31,11 @@ futures-util = { version = "0.3", default-features = false }
 tokio = { version = "1.0", features = ["io-util", "net"] }
 either = "1"
 thiserror = "1.0"
+async-trait = "0.1.81"
 
 [dev-dependencies]
 futures-executor = "0.3"
 tokio = { version = "1.0", features = ["io-util", "rt-multi-thread", "net"] }
 once_cell = "1.2.0"
 hyper = "0.14"
+async-trait = "0.1.81"
```

Comment:
I don't think it's necessary here. No test is using `async-trait`.

Reply:
Yeps, will remove